README.md

```diff
@@ -1,7 +1,50 @@
-# TravIS
-## Google STEP Capstone Project
+# TravIS: Travel Information System
+### Google STEP Capstone Project
 
 This is the Google STEP capstone project by Chris, Adam, and Eshika. 
-**T**ravel **I**nformation **S**ystem is meant to fascilate travel planning.
+TravIS, the **T**ravel **I**nformation **S**ystem, is meant to facilitate travel planning, allowing you to plan the optimal trip.
 
-Visit the website at http://offroad-pod-step-2020.appspot.com.
+Visit the website at http://offroad-pod-step-2020.appspot.com. (Note: This link may become defunct at some point.)
+
+## Main Features
+1. Create a new trip by adding a destination (hotel), start and end dates, and points of interest (POIs) they would like to visit. TravIS will calculate the optimal route and itinerary for the user's trip.
```

Comment:
nit: instead of saying "they", use "the user" to match the style/tense of the rest of the features list

Reply:
Done -- thanks!